flutter_appauth/example/pubspec.yaml

```diff
@@ -14,6 +14,10 @@ dependencies:
     path: ../
   http: ^0.13.0
 
+dependency_overrides:
```

Comment:
Can you remove this? It's not needed as the repo uses [melos](https://melos.invertase.dev)

Reply:
Ah right okay, is this something that's common knowledge in flutter (I've been using dart/flutter for a very short time) or is it worth adding this to documentation?

Reply:
When it comes to Flutter overall, I'd say no. However, those who have been working on plugins/packages with monorepos for a while would likely know. The repo doesn't document so agreed there could be documentation added (e.g. in the contributing guide) but would be out of scope of the PR

Reply:
Agreed! :)